### DISCUSSION TITLE
移动用户

### DISCUSSION THREAD

**Discussion by yezi692** [BearWare/TeamTalk5#1759] [Q&A]
### Description

Give feedback on the problems encountered in the qt version. Qt version, CTRL+alt+x selection, users can't select more than one. According to the conventional barrier-free version, that is, the way of auxiliary functions is not normal. Mobile users must be in the online list, hoping to improve in the next version and keep the original customary operation mode.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by CoBC** [comment]
To select more than one user to move in Qt client, you have to select users in tree like in Windows explorer. So, press shift + arrow keys to select user, or ctrl + arrow keys and space to select, and press CTRL+Alt+X one time.
Hope this helps.
